android/filament-android/src/main/java/com/google/android/filament/View.java

```diff
@@ -310,6 +311,20 @@
         public boolean enabled = false;
     }
 
+    /**
+     * Options to control Depth of Field (DoF) effect in the scene
+     *
+     * @see View#setDepthOfFieldOptions
+     */
+    public static class DepthOfFieldOptions {
+
+        /** focus distance in world units */
+        public float focusDistance = 10.0f;
+
```

Comment:
Should we add a blurScale or for later?

Reply:
I'm not sure where the blur scale would be applied, if we just apply it to the CoC scale and bias, would it be useful? for e.g.: I mean that a scale of 0.5 wouldn't always give the same amount of blur, it would still depend on the camera parameters.  It would be useful for the relative mode though.

Reply:
I was thinking it would be relative to the camera settings. At least you would be able to set your exposure and if you don't like the DoF, the blur scale would let you tweak just the DoF without changing anything else. It's the "break the physically based camera" slider basically.

Reply:
Sounds good.